app/components/markdown/markdown_link.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+import {Linking, Text} from 'react-native';
+
+import CustomPropTypes from 'app/constants/custom_prop_types';
+
+export default class MarkdownLink extends PureComponent {
+    static propTypes = {
+        children: CustomPropTypes.Children.isRequired,
+        href: PropTypes.string.isRequired
+    };
+
+    handlePress = () => {
+        const url = this.props.href;
+
+        Linking.canOpenURL(url).then((supported) => {
```

Comment:
Nothing to do here I think this is ok for now, but we should think about url's like permalinks and how are we going to handle them